docs/client/full-api/api/accounts.md

```diff
@@ -318,7 +318,7 @@ are called with a single argument, the attempt info object:
 
 <dl class="objdesc">
 {{#dtdd name="type" type="String"}}
-  The service name, such as "password" or "twitter".
+  (Server only) The service name, such as "password" or "twitter".
```

Comment:
I'm not sure what these changes are - this is the documentation for `Accounts.validateLoginAttempt`, which isn't related to this PR?